src/libs/analytics.ts

```diff
@@ -0,0 +1,49 @@
+declare global {
+  interface Window {
+    fathom: any;
+  }
+}
+
+export function trackAccordion({ category }) {
+  if (typeof window.fathom === 'undefined') {
+    return;
+  }
+  switch (category) {
+    case 'Accountability':
+      window.fathom.trackGoal('DMTGL7XP', 0);
+      break;
+    case 'Purpose':
+      window.fathom.trackGoal('MZO2GUFJ', 0);
+      break;
+    case 'Technology Type':
+      window.fathom.trackGoal('LZ05MSMC', 0);
+      break;
+    case 'Data Process':
+      window.fathom.trackGoal('C1FPUFGR', 0);
+      break;
+    case 'Data Type':
+      window.fathom.trackGoal('JEBLTBAZ', 0);
+      break;
+    case 'Storage':
+      window.fathom.trackGoal('NF0NX4TA', 0);
+      break;
+    case 'Access':
+      window.fathom.trackGoal('QDGBAC4O', 0);
+      break;
+    default:
+  }
+}
+
+export function trackFAQ() {
+  if (typeof window.fathom === 'undefined') {
+    return;
+  }
+  window.fathom.trackGoal('ODMDAQNP', 0);
+}
+
+export function trackFeedback() {
+  if (typeof window.fathom === 'undefined') {
+    return;
+  }
+  window.fathom.trackGoal('CRROFXAU', 0);
```

Comment:
Not critical, but there's a lot of duplicated logic here, it would make sense to map fathom goal ID's with human readable IDs, such as:

```
const fathomEvents = {
  feedback: 'CRROFXAU',
  faq: 'ODMDAQNP',
  accordion: {
     ... more ID's
  }
}

function TrackEvent(key: string, value = 0) {
  if (typeof window.fathom === 'undefined') {
    return;
  }

  const eventId = fathomEvents[key];

  if (!eventId) { throw Error('Fathom event not found') }

  window.fathom.trackGoal(eventId, value);
}

trackEvent('feedback');
```

This way events can be called generically and the window check wouldn't need to be made repeatedly